src/main.cpp

```diff
@@ -8669,16 +8669,16 @@ int64_t ComputeResearchAccrual(int64_t nTime, std::string cpid, std::string oper
         if(fDebug && !bVerifyingBlock) printf("CRE: No prior block exists...\n");
         if (!AreBinarySuperblocksEnabled(pindexLast->nHeight))
         {
-                if(fDebug && !bVerifyingBlock) printf("CRE: Newbie Stake, Binary SB not enabled, "
-                    "dCurrentMagnitude= %.1f\n", dCurrentMagnitude);
-                return dCurrentMagnitude > 0 ? ((dCurrentMagnitude/100)*COIN) : 0;
+            if(fDebug && !bVerifyingBlock) printf("CRE: Newbie Stake, Binary SB not enabled, "
+                                                  "dCurrentMagnitude= %.1f\n", dCurrentMagnitude);
+            return dCurrentMagnitude > 0 ? ((dCurrentMagnitude/100)*COIN) : 0;
         }
         else
         {
             // New rules - 12-4-2015 - Pay newbie from the moment beacon was sent as long as it is within 6 months old and NN mag > 0 and newbie is in the superblock and their lifetime paid is zero
             // Note: If Magnitude is zero, or researcher is not in superblock, or lifetimepaid > 0, this function returns zero
             int64_t iBeaconTimestamp = BeaconTimeStamp(cpid, true);
-            if (IsLockTimeWithinMinutes(iBeaconTimestamp, 60*24*30*6))
+            if (IsLockTimeWithinMinutes(iBeaconTimestamp, pindexBest->GetBlockTime(), 60*24*30*6))
```

Comment:
:+1:
this was probably the main issue
is pindexBest->GetBlockTime() same as nTime here (should be)